src/shape.rs

```diff
@@ -256,18 +256,10 @@ impl<'a> From<&'a swf::tags::DefineShape> for Shape<'a> {
 
                     continue;
                 }
-                swf::ShapeRecord::StraightEdge(line) => Line {
+                swf::ShapeRecord::Edge(edge) => Line {
```

Comment:
Since this is the only `match` arm, the `continue` above can be removed, and the code after the `match` be moved into this arm.

Reply:
Fixed in 241cd049bdacdb59f4477395fd3e393ef102a651